packages/stores/src/mockStore.ts

```diff
@@ -121,14 +135,16 @@ const mock = (): TStores => {
             cfd_score: 0,
             setCFDScore: jest.fn(),
             getLimits: jest.fn(),
-            has_active_real_account: false,
+            has_active_real_account: true,
```

Comment:
@yaswanth-deriv Please don't change the default value as it will affect test files 🙏🏻

Reply:
Yeah i was testing few of the test cases so i changed it..I reverted the changes now.Thanks